services/git-service-impl/src/test/java/io/fabric8/launcher/service/git/AbstractGitServiceTest.java

```diff
@@ -272,6 +275,11 @@ public void pushFile() throws Exception {
 
         //When: pushing a README.md file
         final Path tempDirectory = tmpFolder.getRoot().toPath();
+        Git repo = Git.init().setDirectory(tempDirectory.toFile()).call();
```

Comment:
Better use try-with-resources here in case something bad happens between this line and the `close()` statement